### PR TITLE
[es_PE] Add RUC

### PR DESCRIPTION
**Pull request by Oxicode**
### What is the reason for this PR?

Generate a REG. UNICO DE CONTRIBUYENTES (RUC) number

- [x] A new feature
- [ ] Fixed an issue (resolve #ID)

### Author's checklist

- [x] Follow the [Contribution Guide](https://github.com/FakerPHP/Faker/blob/main/.github/CONTRIBUTING.md)
- [ ] New features and changes are [documented](https://github.com/FakerPHP/fakerphp.github.io)

### Summary of changes

Se agregó la generación de RUCs para Perú en Persona Natural y Persona Jurídica.
Persona Natural empieza con 10
Persona Jurídica empieza con 20

Referencia: http://www2.sunat.gob.pe/pdt/pdtModulos/independientes/p695/TipoDoc.htm

### Review checklist

- [ ] All checks have passed
- [ ] Changes are approved by maintainer

### CODE REVIEW

File: src/Faker/Provider/es_PE/Company.php [hunk 1]
```diff
@@ -65,4 +65,22 @@ public function bs()
...
+     *
+     * Doesn't include a checksum, as peruvians commonly use only the first
+     * 8 digits.
+     *
+     * @example '10164090581'
+     * @example '20105895410'
+     *
+     * @see http://www2.sunat.gob.pe/pdt/pdtModulos/independientes/p695/TipoDoc.htm
+     */
+    public static function ruc($personaJuridica = true)
```
Comment (pimjansen) [inline src/Faker/Provider/es_PE/Company.php:80]: Can be typed as bool since we support 7.1+ only.

**Review by pimjansen — CHANGES_REQUESTED** [review]

File: src/Faker/Provider/es_PE/Company.php (hunk 1 above)
Comment (bramceulemans) [inline src/Faker/Provider/es_PE/Company.php:80]: Please use English variable names.

File: src/Faker/Provider/es_PE/Person.php [hunk 2]
```diff
@@ -64,7 +64,7 @@ class Person extends \Faker\Provider\Person
         'Nájera', 'Naranjo', 'Narváez', 'Nava', 'Navarrete', 'Navarro', 'Nazario', 'Negrete', 'Negrón', 'Nieto', 'Nieves', 'Niño', 'Noriega', 'Núñez',
         'Ocampo', 'Ocasio', 'Ochoa', 'Ojeda', 'Olivares', 'Olivárez', 'Olivas', 'Olivera', 'Olivo', 'Olmos', 'Olvera', 'Ontiveros', 'Oquendo', 'Ordóñez', 'Orellana', 'Ornelas', 'Orosco', 'Orozco', 'Ortega', 'Ortiz', 'Osorio', 'Otero', 'Ozuna',
         'Pacheco', 'Padilla', 'Páez', 'Palacios', 'Palomino', 'Palomo', 'Pantoja', 'Paredes', 'Parra', 'Partida', 'Patiño', 'Paz', 'Pedraza', 'Pedroza', 'Pelayo', 'Peña', 'Perales', 'Peralta', 'Perea', 'Peres', 'Pérez', 'Pichardo', 'Piña', 'Pineda', 'Pizarro', 'Polanco', 'Ponce', 'Porras', 'Portillo', 'Posada', 'Prado', 'Preciado', 'Prieto', 'Puente', 'Puga', 'Pulido',
-        'Quesada', 'Quezada', 'Quiñones', 'Quiñónez', 'Quintana', 'Quintanilla', 'Quintero', 'Quiroz',
+        'Quesada', 'Quezada', 'Quiñones', 'Quiñónez', 'Quintana', 'Quintanilla', 'Quintero', 'Quiroz', 'Quispe',
```
Comment (bramceulemans) [inline src/Faker/Provider/es_PE/Person.php:67]: Please remove unrelated changes in the `Person` provider.

**Review by bramceulemans — CHANGES_REQUESTED** [review]
Thanks for the contribution. I left some comments.

File: src/Faker/Provider/es_PE/Person.php (hunk 2 above)
Comment (Oxicode) [reply src/Faker/Provider/es_PE/Person.php:67]: "Quispe" es uno de apellidos predominantes en Perú
https://rpp.pe/mundo/latinoamerica/cuales-son-los-apellidos-mas-comunes-en-latinoamerica-noticia-988765

Comment (pimjansen) [reply src/Faker/Provider/es_PE/Person.php:67]: Has nothing to do with this PR so unrelated

Comment (bramceulemans) [reply src/Faker/Provider/es_PE/Person.php:67]: I'd like to not have to translate a comment, and it's not related to this PR.

File: src/Faker/Provider/es_PE/Company.php (hunk 1 above)
Comment (Oxicode) [reply src/Faker/Provider/es_PE/Company.php]: Done

Comment (Oxicode) [reply src/Faker/Provider/es_PE/Company.php]: Done

**Comment by pimjansen** [comment]
I see that the documentation PR is closed by you so please ensure there are docs too

File: src/Faker/Provider/es_PE/Company.php [hunk 3]
```diff
@@ -65,4 +65,24 @@ public function bs()
...
+     * 8 digits.
+     *
+     * @example '10164090581'
+     * @example '20105895410'
+     *
+     * @param bool $isNaturalPerson If set to false or empty, RUC starts with 20 and otherwise with 10
+     *
+     * @see http://www2.sunat.gob.pe/pdt/pdtModulos/independientes/p695/TipoDoc.htm
+     */
+    public static function ruc(bool $isNaturalPerson = false)
```
Comment (bramceulemans) [inline src/Faker/Provider/es_PE/Company.php:82]: Return type?

File: src/Faker/Provider/es_PE/Person.php (hunk 2 above)
Comment (bramceulemans) [inline src/Faker/Provider/es_PE/Person.php:67]:
```suggestion
        'Quesada', 'Quezada', 'Quiñones', 'Quiñónez', 'Quintana', 'Quintanilla', 'Quintero', 'Quiroz',
```

File: src/Faker/Provider/es_PE/Company.php (hunk 3 above)
Comment (Oxicode) [reply src/Faker/Provider/es_PE/Company.php]: done

**Comment by pimjansen** [comment]
Lgtm once we update the docs as well

**Review by pimjansen — APPROVED** [review]

File: src/Faker/Provider/es_PE/Company.php [hunk 4]
```diff
@@ -65,4 +65,24 @@ public function bs()
...
+     * @example '10164090581'
+     * @example '20105895410'
+     *
+     * @param bool $isNaturalPerson If set to false or empty, RUC starts with 20 and otherwise with 10
+     *
+     * @see http://www2.sunat.gob.pe/pdt/pdtModulos/independientes/p695/TipoDoc.htm
+     */
+    public static function ruc(bool $isNaturalPerson = false): string
+    {
+        $prefix = $isNaturalPerson ? 10 : 20;
```
Comment (bramceulemans) [inline src/Faker/Provider/es_PE/Company.php:84]: I would prefer that 10 and 20 are strings instead of numbers. It's more clear.

**Review by bramceulemans — CHANGES_REQUESTED** [review]

**Review by pimjansen — APPROVED** [review]

**Review by bramceulemans — APPROVED** [review]
